lib/ui/viewer/gallery/sharing_hidden_album_warning_widget.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:photos/ente_theme_data.dart';
+
+class SharingHiddenAlbumWarning extends StatelessWidget {
```

Comment:
What was the need to creating separate alertDialog for this? Why can't we reuse the existing AlertDialog/ChoiceDialog etc?
If the design needs to be changed, may be we can create a generate Dialog with the new theme and just call that??

Reply:
This is not a dialog, it is a banner on the hidden folder warning users they're sharing a hidden album. Have changed it to "SharingHiddenItemsBanner" as it now shows up if there are any shared files too.